package.json

```diff
@@ -13,9 +13,9 @@
     "format": "npm run lint -- --fix",
     "build": "grunt",
     "watch": "grunt watch",
-    "package": "grunt package",
+    "package": "npm run build && node ./scripts/package.js",
     "travisnpmpublish": "grunt travisnpmpublish",
-    "electronpublish": "grunt electronpublish"
+    "electronpublish": "node ./scripts/electronpublish.js"
```

Comment:
I also don't particularly like the `electronpublish`-name since it doesn't actually publish anything as far as I can tell.

Maybe we could use `electronzip` or `electronarchive` instead.